snowflake/data_access_sync_to_target.go

```diff
@@ -1661,7 +1661,7 @@ func filterWhoExpression(ap *importer.AccessProvider, roleNameMap map[string]str
 		}
 
 		if len(roles) > 0 {
-			whoExpressionParts = append(whoExpressionParts, fmt.Sprintf("current_role() IN (%s)", strings.Join(roles, ", ")))
+			whoExpressionParts = append(whoExpressionParts, fmt.Sprintf("IS_ROLE_IN_SESSION(%s)", strings.Join(roles, " OR ")))
```

Comment:
For two roles, this would give
`IS_ROLE_IN_SESSION(ROLE1 OR ROLE2)`

That doesn't feel right? Add some tests for multiple roles maybe

Reply:
Oh good catch...